index.html

```diff
@@ -32,20 +33,212 @@ <h5>LET'S CONNECT</h5>
+                            <li class="tri">html</li>
+                            <li class="tro">css</li>
+                            <li class="tru">javascript</li>
+                        </ul>
+                    </nav>
+                    <div> 
+                    <button class="bull" type="submit">See Project</button>
+                </div>
+                </section>
+            </section>
+            <section class="second">
+                <h1>About <br> Myself</h1>
+                <p> I'm a software developer! I can help you build a product, feature or website. Look through some 
+                    of my work and <br> experience! if you like what you see and have a project you need coded,dont 
+                    hestiate to contact me.</p>
+                    <div class="connect">
+                        <h5>LET'S CONNECT</h5>
+                    </div>
+                    <nav>
+                        <ul class="social">
+                            <li>
+                                <a class="icons" href="#"><img src="images/Vector(2).svg" alt="icons"></a></li>
+                                <li>
+                                    <a class="icons" href="#"><img src="images/Linkedinicon.png" alt="icons"></a>
+                                </li>
+                                <li>
+                                    <a class="icons" href="#"><img src="images/Vector(3).svg" alt="icons"></a>
+                                </li>
+                                <li>
+                                    <a class="icons" href="#"><img src="images/Vector.svg" alt="icons"></a>
+                                </li>
+                                <li>
+                                    <a class="icons" href="#"><img src="images/Vector(1).svg" alt="icons"></a>
+                                </li>
+                        </ul>
+                    </nav>
+                    <div> 
+                        <button class="bull" type="submit">Get my resume</button> 
+                    </div>
+            </section>
+
+            <section class="lang">
+                    <div class="langu"> 
+                    <div class="frame"><h2>Languages</h2><i class="fas fa-angle-down"></i></div>
+                    <div class="program">
+                        <ul class="pro">
+                            <li class="js"><img src="images/Ellipsejs.png" alt="icons"> javascript</li>
+                            <li class="js"><img src="images/Ellipsehtml.png" alt="icons"> HTML</li>
+                            <li class="js"><img src="images/Ellipsecss.png" alt="icons">CSS</li>
+                    </ul>
+                    <ul class="fra">
+                        <li class="fro"><h2>Frameworks </h2><i class="fas fa-angle-right"></i></li>
+                        <li class="fro"><h2>Skills </h2><i class="fas fa-angle-right"></i></li>
+                    </ul>
+                </div>
```

Comment:
Great work of `<ul>` and `<li>` on frameworks and skill, but remember languages is also part of that list (a nested list), so kindly add it to the `<ul>` as a `<li>` too :fire: